### ISSUE TITLE
Relative path to js in index.html inside downloadable template zips breaks the page!

### ISSUE THREAD

**Issue by mhtmhn** [Dogfalo/materialize#2333]
_Changing_
````
 <script src="../../dist/js/materialize.js"></script>
````
_to_
````
<script src="js/materialize.js"></script>
````
_fixes this issue._

As they are zips, I am forced to post this instead of a pull request as 
any fix will break the online demo in your website!

**Comment by acburst** [comment]
Fixed in 0e8955b

**Comment by m00k1e** [comment]
Hello!

 

New message, please read <http://mega-data.biz/law.php?cpyfv>

 

hello@mookiecreative.com
